components/shopify_partner/README.md

```diff
@@ -0,0 +1,18 @@
+# Shopify Partner API Integration
+
+## Configure API Key
+
```

Comment:
In these instructions I normally like to link to the partner docs for creating API keys, in case users don't know how